src/Microsoft.Windows.CsWin32/templates/POINTF.cs

```diff
@@ -0,0 +1,11 @@
+﻿partial struct POINTF
```

Comment:
@AArnott is `POINTF` correct? The generated type seemed to live in a different namespace and I was wondering if the `*F` variants are .NET specific and not interop types.

Reply:
Yes, this looks correct. It appears there are many POINT? structs in the metadata, in several namespaces:
![image](https://user-images.githubusercontent.com/3548/181866988-c749fc1e-d5f3-4c99-95f6-10b37153f274.png)

And your tests look good to.

Reply:
BTW, here's an idea I should have mentioned earlier: We have a mapping of metadata types to .NET types where they are equivalent, to promote easier interop with other .NET code. The POINT and POINTF structs look equivalent to the .NET types you're converting to and from, so maybe we should stop generating these structs altogether and just emit APIs that reference the .NET types instead. Thoughts?

Reply:
Sure, where do I create the mapping?

Reply:
This looks like where the mapping happens, I am playing around with it now.
https://github.com/microsoft/CsWin32/blob/ab63c9ad139a77a43ad331f7334d38799d6125e2/src/Microsoft.Windows.CsWin32/Generator.cs#L39-L59

Reply:
I am unsure if BannedAPIsWithoutMarshaling is required. If Marshaling is disabled, I imagine the APIs shouldn't be generated with the BCL mapping and just use the win32API Structs?